rentmyjunk/src/components/RegisterForm.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Form from "./Form";
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+class RegisterForm extends Form {
+    render() {
+        return (
+            <div class="container center_div">
+            <h1 align="center"><font face="Trebuchet MS"><b>Create An Account</b></font></h1>
+            <form action="/action_page.php">
+                <div class="form-group">
+                    <label for="usrname">Username:</label>
```

Comment:
There is a typo on line 13 and 14. Missing e in username.
Line 29 use Environment variables instead of publicizing the API key.
                    <div class="g-recaptcha" data-sitekey=CAPTCHA_KEY></div>